### DISCUSSION TITLE
Welcome to Energy Assistant Discussions!

### DISCUSSION THREAD

**Discussion by pail23** [pail23/energy-assistant-backend#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nrhein** [comment]
Hi, I am missing the information on the benefit of Energy Assistant. What exactly can you do with it, which can't be done by EMHASS or evcc?
